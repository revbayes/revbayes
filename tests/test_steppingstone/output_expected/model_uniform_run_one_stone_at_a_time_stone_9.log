Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-36113.41	-36115.84	2.435569	3.552501	0.223252	0.0387112	0.0379769	0.10544	0.0323261	0.110213	0.455625	0.0145043	0.0997645	0.0416237	0.198209	0.0425095	0.18637	0.0401509	0.0763306	0.166296	0.00949165	0.172876	0.0712205	0.139621	0.162053	0.165441	0.30709	0.00227635	0.15681	0.0277499	0.0869872	0.016484	0.195186	0.082728	0.00515093	0.0221139	0.0542201	0.0740903	0.0512667	0.143244	0.0964886	0.0745043	0.0631808	0.0179345	0.0188396	0.0516701	0.180617	0.0173478	0.202211	0.41083	0.124861	0.228667	0.0160842	0.361636	0.539263	0.0488556	0.0502455	0.8589974	4.318636
10	-31851.11	-31853.4	2.286856	3.024354	0.177248	0.066381	0.00962151	0.0432755	0.0290136	0.0708355	0.119382	0.0186249	0.125698	0.182762	0.0999358	0.0653566	0.0674621	0.052716	0.0691336	0.361907	0.0128021	0.312548	0.0669456	0.0654977	0.0743808	0.197938	0.134014	0.00364328	0.0977715	0.114631	0.133628	0.0132073	0.148296	0.511656	0.00748397	0.0247911	0.138196	0.0309647	0.0909673	0.132207	0.169648	0.0586713	0.0452459	0.0154446	0.0195582	0.0422606	0.150895	0.0196543	0.190131	0.240108	0.0306655	0.507345	0.0120961	0.276361	0.531165	0.121911	0.0705627	0.8468854	4.372648
