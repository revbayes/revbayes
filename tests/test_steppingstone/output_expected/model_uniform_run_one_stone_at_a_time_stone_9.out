state	power	likelihood
4	1e-05	-30043.5
5	1e-05	-32675.6
6	1e-05	-31524.3
7	1e-05	-33528.1
8	1e-05	-30590.3
9	1e-05	-31383.5
10	1e-05	-33182.8
